client/src/components/Kanban/CardForm/CardForm.tsx

```diff
@@ -29,7 +30,7 @@ const CardForm = ({ columnId }: CardFormProps): JSX.Element => {
 
   return (
     <Box>
-      {formIsOpen ? <InnerForm columnId={columnId} formAction={setFormIsOpen} /> : null}
+      {formIsOpen && <InnerForm columnId={columnId} formAction={setFormIsOpen} />}
```

Comment:
@street-jackal using syntactic sugar here; instead of a ternary with a condition (i.e., `formIsOpen ? <InnerForm columnId={columnId} formAction={setFormIsOpen} /> : null`), you can do `formIsOpen && <InnerForm columnId={columnId} formAction={setFormIsOpen} />` to signify "if formIsOpen, display the component. Otherwise, don't do anything"

Reply:
I've seen this syntax before but never known what it meant. Also, I've probably written several lines like this myself (though I didn't write this one!). Good tip!